subsys/flash_map/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+zephyr_sources_ifdef(CONFIG_FLASH_MAP flash_map.c)
+
+if(CONFIG_FLASH_MAP)
+zephyr_sources_ifndef(CONFIG_FLASH_MAP_CUSTOM flash_map_default.c)
```

Comment:
Indentation.

Reply:
You have alread ifdef'd this entire directory, so you dont' need to ifdef flash_map again here. The same applies to FCB I assume.